client/lib/popup.js

```diff
@@ -161,6 +161,14 @@ window.Popup = new class {
     const $element = $(element);
     return () => {
       windowResizeDep.depend();
+
+      if(Utils.isMobile()) {
+        $('.pop-over').addClass('full');
```

Comment:
Instead of imperative `addClass` and `removeClass` I would expose the `isMobile` variable to Blaze templates and use something like 

```jade
.pop-over(class="{{#if isMobile}}full{{/if}}")
```